tox.ini

```diff
@@ -18,8 +18,8 @@ deps =
     requests
     bottle >= 0.10
     flask >= 0.10
-    sqlalchemy
-    Flask-SQLAlchemy
+    sqlalchemy==1.3.*
```

Comment:
I think we should only pin for py 3.4 and 3.5 and should still test with the latest version of sqlalchemy and flask-sqlalchemy for the supported python. Ideally, the SDK should work with every sqlalchemy version supported for each python version.

Reply:
The other py SDKs will [fail](https://github.com/aws/aws-xray-sdk-python/runs/2143938554) the test with the latest sqlalchemy and flask-sqlalchemy version.

Reply:
Why does the X-Ray SDK tests fail for the latest version of sqlalchemy? we are not using any of the internal APIs of sqlalchemy in our tests right? Does that mean the SDK is not compatible with latest sqlalchemy for supported python versions?

Reply:
Some of the unit tests are using the [API](https://github.com/aws/aws-xray-sdk-python/blob/master/tests/ext/sqlalchemy/test_query.py#L62)(for example), which will fail the test for the latest sqlalchemy and flask-sqlalchemy. Not sure if the Python SDK is compatible with the latest sqlalchemy or not, but at least some unit tests need to update the API they use from sqlalchemy and flask-sqlalchemy.